docs/source/understanding_api.md

```diff
@@ -0,0 +1,46 @@
+# Understanding the API Documentation
+
+This page aims to help ShapePipe users and developers understand the
+Application Programming Interface (API) documentation.
+
+```{note}
+If you are already familiar with this type of documentation you can skip this page.
+```
+
+## What Are API Docs?
+
+The API documentation is designed to communicate in clear way what each class
+and function in the package does. For example, what inputs they expect, what
+outputs they provide, and what the various options do. These pages are
+automatically generated from docstrings (i.e. the sections starting and ending with three double quotes `"""`) written in the code.
+
+## Standard API Docs
+
+All the classes/functions include a short description of what they do. This is followed by a `Parameters` section containing a bullet point list of the expected input arguments. For each parameter you will see in brackets the expected input type (e.g. `int`, `float`, `list`, etc.) followed by a brief description of what the argument is for. Parameters listed as *optional* in the brackets do not need to be provided and will default to some predefined value.
+
+```{note}
+If an optional argument does not explicitly specify the default parameter value then the user should expect that this means the default will be `None`, `''`, `[]`, etc. depending on the input data type.
```

Comment:
this renders very weirdly, and the reader has to scroll horizontally to read the entire sentence.

Reply:
OK, should be OK now